letsencrypt-auto

```diff
@@ -61,10 +58,34 @@ if test "`id -u`" -ne "0" ; then
       done
       su root -c "$args"
     }
-    SUDO=su_sudo
-  fi
+}
+
+if /usr/bin/env | grep '^SUDO=' 1>/dev/null 2>&1; then
+  case "$SUDO" in
```

Comment:
I'm happy to change to another variable name. Should I create a new pull request?